src/internal/connector/onedrive/common/consts.go

```diff
@@ -0,0 +1,13 @@
+package common
+
+import "strings"
+
+const (
+	MetaFileSuffix    = ".meta"
+	DirMetaFileSuffix = ".dirmeta"
+	DataFileSuffix    = ".data"
+)
+
+func IsMetaFile(name string) bool {
```

Comment:
This feels more like a `hasMetaSuffix` than an `isMetaFile` check.   To avoid gotchas, I think we should move the graph/metadata func into this file and have a singular owner, rather than split ownership.

Reply:
the one in `graph/metadata` is used by kopia because we wanted to filter out metadata files from details when merging for incrementals. We could probably scrap that code (in another PR) and remove that function completely if we wanted to. Unfortunately though, we'll probably always need something that filters out metadata entries from backup details because folks can still download old releases that had that in it

Reply:
So, contrary to the other comment, that does sound like a service outside of onedrive has the notion of metadata.

Reply:
> Again, there's also nothing stopping folks from downloading an old corso binary that generates metadata details entries and running that

We should keep these things in mind when we look toward a v1 release.  That will allow us some leeway in dictating what is and isn't portable across versions.